### DISCUSSION TITLE
Ying-Jung (Helen) Deweese

### DISCUSSION THREAD

**Discussion by hydrogeohc** [aiqc/AIQC#57] [Introduction]
Hi Team, 

I am Ying-Jung Deweese. I was a hydrologist with many experiences for handling spatial-temporal data. Currently, I am doing online CS degree. I am hoping to contribute to the time series projects. Thanks :) !

**Comment by aiqc** [comment]
Hey Ying-Jung. Do you any cool spatial-temporal visualizations you can share?
